features/multi_candidate_consumer.feature

```diff
@@ -0,0 +1,61 @@
+Feature: Support multi-candidate consumer
+
+  Scenario: A consumer can publish to a single candidate
+    Given the Consumer owned by groovy with Consumer Token e0bf422d63d65ef1f4fe573a0d461d695edef45a541f07f3747ad37188329368 for candidate groovy
+    And the header Consumer-Key 5f202e7ab75f00af194c61cc07ae6b0c is presented
+    And the header Consumer-Token e0bf422d63d65ef1f4fe573a0d461d695edef45a541f07f3747ad37188329368 is presented
+    And the remote release service returns a CREATED response:
```

Comment:
What call is performed here?

Reply:
It can be anything, the response is simply proxied from the remote service.